docs/generate-versions.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+for version in ["0.7", "0.8", "0.9"]:
```

Comment:
Could we parse the versions (> minimum supported version) from build file ? In that way, we dont have to change this file whenever there is a new version released